### PR TITLE
feature: Added connection to supabase

### PR DESCRIPTION
**Pull request by lobarzanek**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SUPABASE_URL = paste here your supabase url
+SUPABASE_KEY = paste here your supabase key
```
Comment (kamilziemann) [inline .env]: Please add `.env` to `.gitignore`, and create `.env.example` with:
```
SUPABASE_URL
SUPABASE_KEY
```

File: components/Header/SummariesHeader.vue [hunk 2]
```diff
@@ -1,31 +1,18 @@
...
-    products.value = data[0].length;
-    orders.value = data[1].length;
-    customers.value = data[2].length;
-  } catch (error) {
-    console.error("There was an error fetching the data", error);
-  }
+  const response = await supabase.from("header_details").select("*");
+  products.value = response.data[0].products_number;
+  orders.value = response.data[0].orders_number;
+  customers.value = response.data[0].users_number;
```
Comment (kamilziemann) [inline components/Header/SummariesHeader.vue]:
```suggestion
  const responseData = response.data[0]
  products.value = responseData.products_number;
  orders.value = responseData.orders_number;
  customers.value = responseData.users_number;
```

Comment (kamilziemann) [reply components/Header/SummariesHeader.vue]: Or:
```
  const { products_number, orders_number, users_number }= response.data[0] ?? {}
  products.value = products_number;
  orders.value = orders_number;
  customers.value = users_number;
  ```

**Review by kamilziemann — APPROVED** [review]

File: .env (hunk 1 above)
Comment (lobarzanek) [reply .env]: done

File: components/Header/SummariesHeader.vue (hunk 2 above)
Comment (lobarzanek) [reply components/Header/SummariesHeader.vue]: done

**Review by kamilziemann — APPROVED** [review]

**Pull request merged**